PayPalNativeCheckout/src/test/java/com/braintreepayments/api/PayPalNativeCheckoutClientUnitTest.java

```diff
@@ -25,17 +25,24 @@
 
 import org.json.JSONException;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.ArgumentCaptor;
 import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
 import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.modules.junit4.rule.PowerMockRule;
+import org.robolectric.RobolectricTestRunner;
 
-@RunWith(PowerMockRunner.class)
-@PrepareForTest( { PayPalCheckout.class, PLog.class })
+@RunWith(RobolectricTestRunner.class)
+@PowerMockIgnore({"org.powermock.*", "org.mockito.*", "org.robolectric.*", "android.*", "androidx.*"})
+@PrepareForTest({PayPalCheckout.class, PLog.class})
 public class PayPalNativeCheckoutClientUnitTest {
 
+    @Rule
+    public PowerMockRule powerMockRule = new PowerMockRule();
```

Comment:
Why was this added to PayPal Native tests? Doesn't look like any of the tests were updated to use this new powerMock rule?

Reply:
We added it so we could run Robolectric and PowerMock at the same time. There's a slight difference in the internals that caused Android code to be called and this test blew up because it wasn't mocked.